### PR TITLE
feat(frontend): add missing caldav translation

### PR DESCRIPTION
**Pull request by gbonaspetti**
#5530 

## Checkpoints

**For developers**

- [x] If relevant, manual tests have been done to ensure the stability of the whole application and that the involved feature works
- [x] The original issue is up to date w.r.t the latest discussions and contains a short summary of the implemented solution
- [x] Automated tests covering the feature or the fix, have been written, deemed irrelevant (give the reason), or an issue has been created to implement the test (give the link)
- [x] Make sure that:
  - if there are modifications in the Tracim configuration files (eg. `development.ini`), they are documented in `backend/doc/setting.md`
  - any migration process required for existing instances is documented
  - relevant people for these changes are notified
- [x] Original authors of the features included in a multi-feature branch (maintenance fixes -> develop, security fixes -> develop, …) should be part of the reviewers, especially if you encountered merge conflicts.

**For code reviewers**

- [x] The code is clear enough
- [x] If there are FIXMEs in the code, related issues are mentioned in the FIXME
- [x] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present

**For testers**

- [x] Manual, quality tests have been done

### CODE REVIEW

File: frontend/dist/assets/_caldavzap/localization.js [hunk 1]
```diff
@@ -1127,14 +1127,14 @@ var localizationCalDAV = new Object();
 
 localizationCalDAV['ar']=	/* thanks Myriam Rabhi */
 {
-	_default_title_format_month_: 'yyyy MMM',
+	_default_title_format_month_: 'yyyyy MMM',
```
Comment (gbonaspetti) [inline frontend/dist/assets/_caldavzap/localization.js:1130]: I think it has an extra `y`. An year is max 4 y.

File: frontend/dist/assets/_caldavzap/localization.js [hunk 2]
```diff
@@ -1360,13 +1360,13 @@ localizationCalDAV['ar']=	/* thanks Myriam Rabhi */
...
+	loadingCalendars: 'تحميل التقويمات (%act% من %total%) …',
+	unloadingCalendars: 'إلغاء تحميل التقويمات...',
 
-	errUnableSaveBeginCalDAV: "خطأ: \'تسجيل المناسبة غير ممكن\'%%!",
-	errUnableDeleteBeginCalDAV: "خطأ: \'حذف المناسبة غير ممكن/ :%%!",
+	errUnableSaveBeginCalDAV: "خطأ: 'تسجيل الفعالية غير ممكن': %%!",
+	errUnableDeleteBeginCalDAV: "خطأ: 'حذف الفعالية غير ممكن' : %%!",
 	CalDAVerrHttp410: 'المناسبة لم تعد موجودة على الخادم',
-	errUnableSaveTodoBeginCalDAV: "خطأ: \'حفظ المهمة غير ممكن\' :%%!",
+	errUnableSaveTodoBeginCalDAV: "خطأ: 'حفظ المهمة غير ممكن' : %%!",
```
Comment (gbonaspetti) [inline frontend/dist/assets/_caldavzap/localization.js:1369]: don't you have to escape the `'`?

Comment (knodelle) [reply frontend/dist/assets/_caldavzap/localization.js]: I fixed this

**Review by gbonaspetti — COMMENTED** [review]
Approved (I can't do it because I created the PR)

**Review by knodelle — APPROVED** [review]

**Comment by gbonaspetti** [comment]
Tested on branch

**Pull request merged**